Doc/tutorial/introduction.rst

```diff
@@ -478,73 +478,123 @@ example::
-  anything with a non-zero length is true, empty sequences are false.  The test
-  used in the example is a simple comparison.  The standard comparison operators
-  are written the same as in C: ``<`` (less than), ``>`` (greater than), ``==``
-  (equal to), ``<=`` (less than or equal to), ``>=`` (greater than or equal to)
-  and ``!=`` (not equal to).
-
-* The *body* of the loop is *indented*: indentation is Python's way of grouping
-  statements.  At the interactive prompt, you have to type a tab or space(s) for
-  each indented line.  In practice you will prepare more complicated input
-  for Python with a text editor; all decent text editors have an auto-indent
-  facility.  When a compound statement is entered interactively, it must be
-  followed by a blank line to indicate completion (since the parser cannot
-  guess when you have typed the last line).  Note that each line within a basic
-  block must be indented by the same amount.
-
-* The :func:`print` function writes the value of the argument(s) it is given.
-  It differs from just writing the expression you want to write (as we did
-  earlier in the calculator examples) in the way it handles multiple arguments,
-  floating point quantities, and strings.  Strings are printed without quotes,
-  and a space is inserted between items, so you can format things nicely, like
-  this::
-
-     >>> i = 256*256
-     >>> print('The value of i is', i)
-     The value of i is 65536
-
-  The keyword argument *end* can be used to avoid the newline after the output,
-  or end the output with a different string::
-
-     >>> a, b = 0, 1
-     >>> while a < 1000:
-     ...     print(a, end=',')
-     ...     a, b = b, a+b
-     ...
-     0,1,1,2,3,5,8,13,21,34,55,89,144,233,377,610,987,
+   4
 
+Function argumets
+-----------------
+
+We already know :func:`print` function, that writes the value of the
+argument(s) it receives on screen.  The arguments are enclosed within
+parentheses ``()``. In simplest form, like ``print(a, b)``, the arguments
+are positional,  meaning the function processes them in the same order
+as they are written::
+
+   >>> group = "Knights"
+   >>> say_what = '"Ni!"'
+   >>> print(group, 'Who Say', say_what)
+   Knights Who Say "Ni!"
+
+The ``print()`` function also offers a useful keyword argument called *end*,
+which can be used to avoid the newline after the output, or end the output
+with a different string::
+
+   >>> count = 0
+   >>> count = 10
+   >>> while count > 5:
+   ...     print(count, end=' ')  # end the output of print() with a whitespace
+   ...     count = count - 1  # lower the value by one with each repetition
+   ...
+   10 9 8 7 6
+
+Fibonacci series
+----------------
+
+Great!  Now that we have the knowledge, let's write code to print all numbers
+from the Fibonacci series that are lower than 150::
+
+   >>> a, b = 0, 1
+   >>> while a < 150:
+   ...      print('', a, '', end='->')
```

Comment:
I understand that this utilities the print sep param, however this is nor explained nor inferred, so to a beginner it may be confusing 

Perhaps it could be beneficial to show off f-strings like 
```suggestion
   ...      print(f'{a} ->', end='')
```
Or to keep end as `->`
```suggestion
   ...      print(f'{a} ', end="->")
```

Reply:
I very strongly prefer the original `print('', a, '', end='->')`, which reduces the punctuation the learner has to deal with to a minimum, and emphasises instead the connection between more easily assimilated things, like the English word "end" and the "arrow" ("->"). 

Showing off f-strings while demonstrating a different functionality introduces unnecessary cognitive burden. Two different kinds of quotes and two different kinds of brackets requires a lot of parsing and stopping. I don't think this is the right solution here.

Reply:
My reasoning: As this is an Introduction chapter, I wanted to keep the number of introduced concepts to the minimum. There are ways to code this better, but it is not formally wrong. The default separator is already shown in the example above and this just builds on that, so it's hopefully not confusing to the reader.